tensorflow/tools/dist_test/local_test.sh

```diff
@@ -119,10 +125,32 @@ if [[ ! -z "${TF_DIST_DOCKER_NO_CACHE}" ]] &&
   NO_CACHE_FLAG="--no-cache"
 fi
 
+
+if [[ -z "${CUSTOM_WHL_URL}" ]]; then
+  BUILD_DIR="${DIR}"
+else
+  # Custom whl file URL is specified. Modify the Dockerfile before building
+  # the docker image.
+  BUILD_DIR=$(mktemp -d)
+  echo ""
+  echo "Using custom whl file URL: ${CUSTOM_WHL_URL}"
+  echo "Building in temporary directory: ${BUILD_DIR}"
+
+  cp -r ${DIR}/* ${BUILD_DIR}/ || \
+    die "Failed to copy files to ${BUILD_DIR}"
+
+  sed -i -r -e "s@https:\/\/.*\.whl@${CUSTOM_WHL_URL}@g" \
```

Comment:
Instead of this, can we download/copy the wheel file to a folder, and mount that folder in the dockerfile?
This usually leads to confusion when debugging the test, as the engineer cannot see the actual file being executed.